.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+** Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
+
+<!--
```

Comment:
That's why it's a comment. It's meant to give clarification to the person that's opening the issue without adding additional noise to the issue itself.